formspree/templates/static_pages/index.html

```diff
@@ -253,15 +253,15 @@ <h3>Invisible emails</h3>
       <div class="container narrow block">
             <div class="col-1-1">
                 <h3>Unlimited form submissions</h3>
-                <p>Getting more than 1000 submissions each month? No problem, upgrading will cover that.</p>
+                <p>Getting more than {{ config.MONTHLY_SUBMISSIONS_LIMIT }} submissions each month? No problem, upgrading will cover that.</p>
             </div>
       </div>
 
       <div class="container narrow block">
             <div class="col-1-1">
                 <h3>Access to submissions archive</h3>
-                <p>If you ever missed a submission email, for any reason, you can just login and read your last 1000 submissions, all listed and timestamped with full data. Also, the submissions can be exported to CSV or JSON!</p>
-                <p>It even works for submissions made while you were not a {{ config.UPGRADED_PLAN_NAME }} user (the last 100, in this case).</p>
+                <p>If you ever missed a submission email, for any reason, you can just login and read your last {{ config.ARCHIVED_SUBMISSIONS_LIMIT }} submissions, all listed and timestamped with full data. Also, the submissions can be exported to CSV or JSON!</p>
+                <p>It even works for submissions made while you were not a {{ config.UPGRADED_PLAN_NAME }} user (the last {{ config.MONTHLY_SUBMISSIONS_LIMIT }}, in this case).</p>
```

Comment:
We should remove `(the last {{ config.MONTHLY_SUBMISSIONS_LIMIT }}, in this case)`. If we only store the monthly limit for non-gold users then they have no reason to upgrade, because submissions over the limit are discarded anyway.